src/spring/azext_spring/tests/latest/test_asa_application_configuration_service.py

```diff
@@ -91,5 +91,17 @@ def test_application_configuration_service(self, resource_group, spring, app):
             self.check('properties.provisioningState', "Succeeded")
         ])
 
-        self.cmd('spring application-configuration-service update -g {rg} -s {serviceName} --generation Gen2',
-                 checks=[self.check('properties.provisioningState', "Succeeded")])
+        self.cmd('spring application-configuration-service update -g {rg} -s {serviceName} '
```

Comment:
Please add some UnitTest to validate the command code. You can refer the `TestSpringCloudCreateEnterprise` inside the `test_asa_create.py` file.